### ISSUE TITLE
Should SQL Explorer ignore the METL prefix tables by default instead of SYM tables?

### ISSUE THREAD

**Issue by abrougher** [JumpMind/metl#176]

**Comment by chenson42** [comment]
We should at least clear the setting out.  Doesn't make sense for Metl.  All the Metl tables are in the metl db so they are pretty segregated.  Probably don't want to hide them.

**Comment by chenson42** [comment]
I don't think so, but we could remove the SYM_ expression.  The only time you see METL tables is when you are browsing the METL database.  If you are doing that, then you probably want to see them.
